sandpack-react/src/components/CodeEditor/CodeMirror.tsx

```diff
@@ -185,6 +195,54 @@ export const CodeMirror: React.FC<CodeMirrorProps> = ({
     }
   }, [code]);
 
+  React.useEffect(
+    function messageToInlineError() {
+      if (showInlineErrors) {
```

Comment:
nitpick: I'd revert the condition here to get rid of the extra indentation level